mne/tests/test_epochs.py

```diff
@@ -76,6 +76,14 @@ def test_reject():
                   picks=picks, preload=False, reject='foo')
     assert_raises(ValueError, Epochs, raw, events, event_id, tmin, tmax,
                   picks=picks_meg, preload=False, reject=dict(eeg=1.))
+    with warnings.catch_warnings(record=True) as w:
+        Epochs(raw, events, event_id, tmin, tmax, picks=picks_meg,
+               preload=False, reject=dict(eeg=1.),
+               allow_missing_reject_keys=True)
```

Comment:
for compatibility, I suppose we should also add this to `drop_bad_epochs`? It would be weird to have it only in the `Epochs` constructor ...